### ISSUE TITLE
Receive emails when followed crates publish a new release

### ISSUE THREAD

**Issue by rk3141** [rust-lang/crates.io#3081]
**Description**
I would want to be able to star/pin/follow crates and receive emails regarding updates.

Sorry if this isn't clear, I am not good at describing ideas :|

**Comment by Turbo87** [comment]
are you aware that we already have a system to follow crate updates when you're logged in? or is the main request here that we should send those update emails?

**Comment by rishit-khandelwal** [comment]
The email part.

**Comment by LikeLakers2** [comment]
Although I imagine that it'll be implemented anyways, I'd like to ask that it be made optional if this is implemented -- and honestly, it should probably default to off. Personally, I use the follow button as a sort of bookmarking system, marking crates I'm interested in using, but I would rather not get emails about that. Beyond that, it may be odd if someone started receiving emails from crates.io out of the blue!

**Comment by extrawurst** [comment]
I very much would also like to have this feature. maybe even the option to have a daily digest instead of adhoc emails right when a release arrived. 

the problem is that a lot of authors do not align git tags with crates releases so it is sometimes hard to follow all crates releases of packages I am interested in.

> and it should probably default to off.

I would argue to make it off for existing users and opt-out for new users. but that is a matter of taste.

**Comment by Turbo87** [comment]
with https://github.com/rust-lang/crates.io/pull/9064 we now have per-crate RSS feeds, that can be followed as desired. while those are not emails, they achieve a similar result. we are unlikely to implement email notifications for releases in the near future.

**Comment by extrawurst** [comment]
@Turbo87 what holds us back from doing emails? Would love to talk about that - or a pwa with native notifications?

**Comment by Turbo87** [comment]
the extra complexity and the fact that we have a limited budget for sending emails
